### PR TITLE
Check Docker image on PR

### PR DESCRIPTION
**Pull request by mpizenberg**

### CODE REVIEW

**Comment by mpizenberg** [comment]
This is to check that Dockerfile builds successfully on PRs. Could be useful to other tracks as well.

**Comment by mpizenberg** [comment]
@ErikSchierboom I'm using v2 of docker/build-push-action for this. Would you like me to try update PR #10 with v2 also before merging it.

File: .github/workflows/pullrequest.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Build Docker image
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          tags: latest
```
Comment (ErikSchierboom) [inline .github/workflows/pullrequest.yml:26]: Looking at [the docs](https://github.com/docker/build-push-action#leverage-github-cache), they have two additional lines here to use the caching:

```suggestion
          push: false
          tags: latest
          cache-from: type=local,src=/tmp/.buildx-cache
          cache-to: type=local,dest=/tmp/.buildx-cache
```

Is that something useful to add?

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
> @ErikSchierboom I'm using v2 of docker/build-push-action for this. Would you like me to try update PR #10 with v2 also before merging it.

That would be great!

**Pull request merged**